src/Microsoft.ML.Core/Data/ColumnTypeExtensions.cs

```diff
@@ -17,5 +17,10 @@ internal static class ColumnTypeExtensions
         public static bool IsStandardScalar(this ColumnType columnType) =>
```

Comment:
>public [](start = 8, length = 6)

Checking, those are public, rather than internal to spare the [BestFriend] decoration?

Reply:
They're definitely internal. The class is internal and `[BestFriend]`ed, so the methods of course don't have to be.